index.html

```diff
@@ -0,0 +1,165 @@
+                    <i class="fa-solid fa-cloud"></i>
+                     <h3>Cloud</h3>
+                
+                    <p>The cloud" refers to servers that are accessed over the Internet, and the software and databases that run on those servers</p>
+                </div>
+            </div>
+
+            <div class="box2">
+                <div class="wrap">
+                    <i class="fa-brands fa-node"></i>
+                     <h3>Node JS</h3>
+                
+                    <p>The cloud" refers to servers that are accessed over the Internet, and the software and databases that run on those servers</p>
+                </div>
+            </div>
+
+            <div class="box3">
+                <div class="wrap">
+                    <i class="fa-brands fa-docker"></i>
+                     <h3>Docker</h3>
+                     <p>The cloud" refers to servers that are accessed over the Internet, and the software and databases that run on those servers</p>
+                </div>
+            </div>
+           
+
+            <div class="box4">
+                <div class="wrap">
+                    <i class="fa-solid fa-database"></i>
+                     <h3>DataBase</h3>
+                     <p>The cloud" refers to servers that are accessed over the Internet, and the software and databases that run on those servers</p>
+                </div>
+            </div>
+
+            <div class="box5">
+                <div class="wrap">
+                    <i class="fa-solid fa-code-branch"></i>
+                     <h3>Grapghs</h3>
+                     <p>The cloud" refers to servers that are accessed over the Internet, and the software and databases that run on those servers</p>
+                </div>
+            </div>
+        </section>
+            <a class="link-program">SEE THE WHOLE PROGRAM</a>
+            <button class="button">Join our Crash Summer Course</button>
+    </section>
+
+
+    <section class="speakers">
+        <h3>Featured Speakers</h3>
+        <hr>
+
+        <div class="speakers-elt">
+        
+        </div>
+            <!-- <div class="more">MORE <img src="style/images/arrow-down.png" alt=""></div> -->
+    </section>
+</div>
+  
+
+    <footer>
+        <div class="partner">
+                <h2>Partner</h2>
+                <hr>
+
+            <div class="partner-corporation">
+                    <h3><i class="fab fa-google"></i>Google</h3>
+                    
+                    <h3><i class="fa-brands fa-linux"></i>Linux</h3>
+                    <h3><i class="fa-brands fa-airbnb"></i>airbnb</h3>
+                    <h3><i class="fab fa-microsoft"></i>Microsoft</h3>
+                    <h3><i class="fa-brands fa-facebook"></i>FaceBook</h3>
+            </div>  
+        </div>
+
+        <div class="logo-rights">
+            <img src="style/images/logo.png" alt="creative-common">
+            <p>@2023 summer BootCamp Some rights reserved</p>
+        </div>
```

Comment:
- [ ] On the desktop view the logo is very small and the text is too large. Kindly increase the size of the logo and decrease the size of the text so your footer can look more similar to the template and look more professional. 
 - Your design:
 - 
<img width="959" alt="image" src="https://user-images.githubusercontent.com/61919617/203773817-87d1520b-d4a9-4758-a841-750c86228928.png">

- Template:
- 
<img width="853" alt="image" src="https://user-images.githubusercontent.com/61919617/203774006-38d9340f-aa17-4d5d-93e1-28265233dfed.png">